Formula/arangodb.rb

```diff
@@ -23,49 +23,18 @@ class Arangodb < Formula
     cause "Fails with compile errors"
   end
 
-  resource "arangodb2" do
-    url "https://www.arangodb.com/repositories/Source/ArangoDB-2.8.10.tar.gz"
-    sha256 "3a455e9d6093739660ad79bd3369652db79f3dabd9ae02faca1b014c9aa220f4"
-  end
-
-  resource "upgrade" do
-    url "https://www.arangodb.com/repositories/Source/upgrade3-1.0.0.tar.gz"
-    sha256 "965f899685e420530bb3c68ada903c815ebd0aa55e477d6949abba9506574011"
-  end
-
   def install
     ENV.cxx11
 
-    (libexec/"arangodb2/bin").install resource("upgrade")
-
-    resource("arangodb2").stage do
-      ENV.cxx11
-
-      args = %W[
-        --disable-dependency-tracking
-        --prefix=#{libexec}/arangodb2
-        --disable-relative
-        --localstatedir=#{var}
-        --program-suffix=-2.8
-      ]
-
-      if ENV.compiler == "gcc-6"
-        ENV.append "CXXFLAGS", "-O2 -g -fno-delete-null-pointer-checks"
-        inreplace "3rdParty/Makefile.v8", "CXXFLAGS=\"", "CXXFLAGS=\"-fno-delete-null-pointer-checks "
-      end
-
-      system "./configure", *args
-      system "make", "install"
-    end
-
     mkdir "build" do
       args = std_cmake_args + %W[
         -DHOMEBREW=ON
         -DUSE_OPTIMIZE_FOR_ARCHITECTURE=OFF
         -DASM_OPTIMIZATIONS=OFF
         -DCMAKE_INSTALL_DATADIR=#{share}
-        -DETCDIR=#{etc}
-        -DVARDIR=#{var}
+        -DCMAKE_INSTALL_DATAROOTDIR=#{share}
```

Comment:
I think share is correct, because the ArangoDB Makefile adds a package name as well.